src/lightning-stubs/recordPicker/recordPicker.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { LightningElement, api } from 'lwc';
+
+export default class RecordPicker extends LightningElement {
+    @api displayInfo;
+    @api filter;
+    @api label;
+    @api matchingInfo;
+    @api objectApiName;
+    @api placeholder;
+    @api required;
+    @api value;
+    @api variant;
+    @api checkValidity() {}
+    @api reportValidity() {}
+    @api setCustomValidity() {}
```

Comment:
fix: add 
- `focus` method
- `blur` method

Reply:
Good point, those methods were not on the stub in the lightning-components repo 🤦

Reply:
Update: `focus` and `blur` methods are not added to the stub when re-generating stubs on the lightning-components repo. It seems they are not necessary (they're not on other components either).

Reply:
Native method are not needed it seems according to LBC [https://github.com/salesforce-experience-platform-emu/lightning-components/blob/ma[…]/lightning-components-stubs/scripts/generate-lightning-mocks.js](https://github.com/salesforce-experience-platform-emu/lightning-components/blob/master/lightning-components-stubs/scripts/generate-lightning-mocks.js#L193)